userprofile/views.py

```diff
@@ -128,5 +128,44 @@ def get_common_objects(self):
         self.privacy_text = privacy_text
         self.note("privacy_text")
 
+class AccessibilityStatementView(UserProfileView):
```

Comment:
Can you add extra newlines before and after the class (i.e. 2 lines between different classes.